content/en/docs/spinnaker/using-dinghy.md

```diff
@@ -1097,3 +1256,223 @@ Delete all Webhook validations with the `--all` parameter.
 ```bash
 hal armory dinghy webhooksecrets <version control provider> delete --all
 ```
+
+
+## Application updates
+
+The Dinghy service behavior for applications and pipelines are different:
+- Applications: Dinghy creates the application with the configuration initially sent. The application does not get updated after that.
```

Comment:
"The application does not get updated after that, even if the Dinghy file has been changed since the initial read."